puppet/zulip/files/supervisor/conf.d/zulip.conf

```diff
@@ -7,40 +7,20 @@
 ; Note: shell expansion ("~" or "$HOME") is not supported.  Environment
 ; variables can be expanded using this syntax: "%(ENV_HOME)s".
 
-[fcgi-program:zulip-django]
-command=/home/zulip/deployments/current/manage.py runfcgi daemonize=False maxchildren=20 ; the program (relative uses PATH, can take args)
-;process_name=%(program_name)s ; process_name expr (default %(program_name)s)
-;numprocs=1                    ; number of processes copies to start (def 1)
-;directory=/tmp                ; directory to cwd to before exec (def no cwd)
-;umask=022                     ; umask for process (default None)
+[program:zulip-django]
+command=/home/zulip/deployments/current/zulip-venv/bin/uwsgi --ini /etc/zulip/uwsgi.ini
```

Comment:
Oh, I think the problem is that you're hardcoding the python 2 virtualenv path here, and the way we switch to Python 3 for the production tests involves using `#!` lines (there's a script `tools/replace-tarball-shebang` that does this).  So I think we want to make the `command` line here somehow be modified by that script...